cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinSecondaryCategory.java

```diff
@@ -30,6 +30,8 @@
     RESEARCH_REPORT("RAPPORT", "RESEARCH_REPORT"),
     DEGREE_PHD("DRGRADAVH", "DEGREE_PHD"),
     DEGREE_MASTER("MASTERGRADSOPPG", "DEGREE_MASTER"),
+    SECOND_THESIS("HOVEDFAGSOPPGAVE", "SECOND_THESIS"),
```

Comment:
Can you add a description of the second thesis in the cristin ontology file in the field "varbeidunderkatkode"?